hello/hello_child_workflow.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+from dataclasses import dataclass
+
+from temporalio import workflow
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+
+@dataclass
+class ComposeGreetingInput:
+    greeting: str
+    name: str
+
+
+@workflow.defn
+class ComposeGreeting:
+    @workflow.run
+    async def run(self, input: ComposeGreetingInput) -> str:
+        return f"{input.greeting}, {input.name}!"
+
+
+@workflow.defn
+class GreetingWorkflow:
+    @workflow.run
+    async def run(self, name: str) -> str:
```

Comment:
No dataclass??
Kidding, I think we should show that those aren't strictly required.
Same way we don't bother with data classes for return values either.

Reply:
Yeah, I'm being a bit practical in several places. The dataclass-everywhere, while recommended in production, just isn't how quick scripters will do things (and lots of Python people seem to be scripters).